### PR TITLE
Fix equation typo in qaoa.cycle loss Hamiltonian docstring

### PR DESCRIPTION
**Pull request by anthayes92**
As titled.

### CODE REVIEW

**Review by trbromley — APPROVED** [review]
Thanks @anthayes92! Still worth updating the changelog, the documentation section has similar entries for typos.

**Comment by anthayes92** [comment]
@trbromley my editor has auto-formatted the changelog again. Thought it's worth mentioning. If it looks ok to you I'll merge this in.

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -462,7 +462,10 @@ random_mat2 = rng.standard_normal(3, requires_grad=False)
 
 <h3>Documentation</h3>
 
-* Fix typo in the documentation of qml.templates.layers.StronglyEntanglingLayers 
+* Fix typo in the documentation of qml.qaoa.cycle.loss_hamiltonian
+  [(#1396)](https://github.com/PennyLaneAI/pennylane/pull/1396)
+
+* Fix typo in the documentation of qml.templates.layers.StronglyEntanglingLayers
```
Comment (trbromley) [inline .github/CHANGELOG.md:468]:
```suggestion
* Fix typo in the documentation of qml.templates.layers.StronglyEntanglingLayers.
```

File: .github/CHANGELOG.md [hunk 2]
```diff
@@ -462,7 +462,10 @@ random_mat2 = rng.standard_normal(3, requires_grad=False)
 
 <h3>Documentation</h3>
 
-* Fix typo in the documentation of qml.templates.layers.StronglyEntanglingLayers 
+* Fix typo in the documentation of qml.qaoa.cycle.loss_hamiltonian
```
Comment (trbromley) [inline .github/CHANGELOG.md:465]:
```suggestion
* Fix typo in the documentation of qml.qaoa.cycle.loss_hamiltonian.
```

**Review by trbromley — APPROVED** [review]
